docs/syntax-reference.rst

```diff
@@ -838,6 +838,12 @@ Scalar functions
 +------------------------+------------------------------------------------------------+---------------------------------------------------+
 | FLOOR                  |  ``FLOOR(col1)``                                           | The floor of a value                              |
 +------------------------+------------------------------------------------------------+---------------------------------------------------+
+| GEO_DISTANCE           | ``GEODISTANCE(lat1, lon1,                                  | Distance between two lat-lon points, |
```

Comment:
The table syntax is broken because the ending `|` must be aligned to the rest of the table.

Think:

```
| foo         |
| geo         |
```

And not:

```
| foo         |
| geo    |
```

Reply:
ahh - got it. i needed a much wider monitor to actually see this :)  - fixed.